modules/music/addmusic.rb

```diff
@@ -0,0 +1,43 @@
+module AddMusic
+  def add_music
+    publish_date = input_date
+    on_spotify = input_on_spotify
+    music_album = MusicAlbum.new(publish_date: publish_date, on_spotify: on_spotify)
+    @music_albums << music_album
+    genre = add_genre
+    music_album.add_genre(genre)
```

Comment:
- Good job done so far, after implementing the custom setter method, to avoid errors in your code, please change this line to:

```ruby
   music_album.genre = genre
```